pulsar-io/netty/src/main/resources/META-INF/services/pulsar-io.yaml

```diff
@@ -0,0 +1,22 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: netty
```

Comment:
I think we should call the connector tcp.  It will be clearer to the user what this connector does. At the end of the day the user will submit a command as such:
./bin/pulsar-admin source create --name test --destinationTopicName test --source-type tcp

is clearer than: 

./bin/pulsar-admin source create --name test --destinationTopicName test --source-type netty